Assets/Scripts/IUnityService.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public interface IUnityService
```

Comment:
I know I suggested to do it the way this guy from the video about tests, but I just realized we can do it even better.

Name of this interface is general and it might be tempting to add other Unity functions here as well if we ever want to have a proxy for them. It might become messy and it will break Interface Segregation Principle. To be honest, it already does now. So, my suggestion:
```suggestion
public interface IInputService
```